tldw_Server_API/tests/Slides/test_slides_generator.py

```
@@ -84,3 +84,75 @@ def bad_llm_call(api_provider=None, messages=None, **kwargs):
             chunk_size_tokens=None,
             summary_tokens=None,
         )
+
+
+def test_timeline_style_generates_visual_block_and_text_fallback():
+    captured = {}
+
+    def timeline_llm_call(api_provider=None, messages=None, **kwargs):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `timeline_llm_call`

(ANN202)